server/svix-server/src/lib.rs

```diff
@@ -268,6 +270,18 @@ pub fn setup_tracing(cfg: &ConfigurationInner) {
         tracing_opentelemetry::layer().with_tracer(tracer)
     });
 
+    let sentry_guard = sentry::init(sentry::ClientOptions {
```

Comment:
Local dev probably shouldn't hit sentry, only deployed dev, staging, and prod. Right?

Reply:
Could be useful to do this when iterating on adding context data (to see how it looks on the Sentry side).

In that situation, we'd probably want to use a DSN for a garbage/sandbox project, so probably the environment doesn't matter 🤣

Reply:
Sentry gets disabled when the DSN is `None`, so I don't think we need an explicit disable for dev.

Reply:
Anyhow, cool, sounds good. CI is failing, but good otherwise.